src/components/AddBooks.js

```diff
@@ -13,23 +13,21 @@ const AddBooks = (props) => {
   };
 
   return (
-    <form onSubmit={handleSubmit}>
-      <div className="row g-3">
-        <div className="mb-2 mt-3 col-md-6">
-          <label htmlFor="category" className="form-label">
-            Category:
-            <input type="text" className="form-control" id="category" placeholder="Category" name="category" required />
-          </label>
+    <div>
+      <form action="#" className="py-5 container" onSubmit={handleSubmit}>
+        <h3 id="form-title">ADD NEW BOOK</h3>
+        <div className="row gx-5 d-flex">
+          <div className="mb-2 mt-3 col-md-7">
+            <input type="text" id="titles" className="form-control p-3" placeholder="Book Title" name="title" required />
+          </div>
+          <div className="mt-3 col-md-3">
+            <input type="text" id="category" className="form-control p-3" placeholder="Category ▼" name="category" required />
```

Comment:
- [ ] <b>REQUIRED:</b> Category must be a selector not just text input. Kindly address that.